elliottlib/cli/find_bugs_qe_cli.py

```diff
@@ -35,7 +35,7 @@ def find_bugs_qe_cli(runtime: Runtime, noop):
     runtime.initialize()
     find_bugs_obj = FindBugsQE()
     exit_code = 0
-    for b in [runtime.bug_trackers('jira'), runtime.bug_trackers('bugzilla')]:
+    for b in [runtime.get_bug_tracker('jira'), runtime.get_bug_tracker('bugzilla')]:
```

Comment:
We can remove `bugzilla` from our bug lookup commands (find-bugs:qe, find-bugs:sweep), probably better in a different PR

Reply:
Sounds good. If you could open a ticket, I can take a look